src/bridge/test-fs.c

```diff
@@ -722,11 +723,12 @@ test_dir_simple (TestCase *tc,
 
   set_contents (tc->test_path, "Hello!");
 
-  setup_fsdir_channel (tc, tc->test_dir);
+  setup_fsdir_channel (tc, tc->test_dir, FALSE);
 
   event = recv_json (tc);
   g_assert_cmpstr (json_object_get_string_member (event, "event"), ==, "present");
   g_assert_cmpstr (json_object_get_string_member (event, "path"), ==, base);
+  g_assert (json_object_get_int_member (event, "type") == 1);
```

Comment:
FWIW, in the future you would use ```g_assert_cmpint(x, ==, y);``` to produce more readable messages.

But I think as @mvollmer mentioned, this would become a string anyway.